scripts/install_karavi_auth.sh

```diff
@@ -35,7 +35,7 @@ function usage() {
 }
 
 UPGRADE=0
-RPM_VERSION=1.5-1
+RPM_VERSION=${VERSION_TAG}
```

Comment:
will $VERSION_TAG be replaced with the correct version when you build the tar using `make package`

Reply:
Yep. You can see the file (karavi_authorization_1.6.0.tar.gz) built [here](https://osj-sio-03-prd.cec.lab.emc.com/job/Ecosystems_Novus/job/Karavi_Authorization/job/karavi-authorization-e2e-latest/221/console) when you scroll down